### PR TITLE
Remove unnecessary code

### PR DESCRIPTION
**Pull request by sturosier**

### CODE REVIEW

File: admin/class-digital-climate-strike-wordpress-admin.php [hunk 1]
```diff
@@ -100,16 +104,16 @@ private function field_is_set($input, $field ) {
      */
     public function validate($input) {
         $valid = array();
-        $valid['show_digital_strike_widget'] = $this->field_is_set($input, 'show_digital_strike_widget') ? esc_attr($input['show_digital_strike_widget']) : 0;
+        $valid['show_digital_strike_widget'] = $this->field_is_set($input, 'show_digital_strike_widget') ? esc_attr($input['show_digital_strike_widget']) : self::DISABLE;
         $valid['language'] = $this->field_is_set($input, 'language') ? esc_attr($input['language']) : 'en';
-        $valid['cookie_expiration_days'] = $this->field_is_set($input, 'cookie_expiration_days') ? esc_attr($input['cookie_expiration_days']) : 1;
+        $valid['cookie_expiration_days'] = $this->field_is_set($input, 'cookie_expiration_days') ? esc_attr($input['cookie_expiration_days']) : self::ENABLE;
```
Comment (dtoakley) [inline admin/class-digital-climate-strike-wordpress-admin.php]: I know it's an extra line of code, but I think this should have it's own constant for clarity. E.g. DEFAULT_COOKIE_EXPIRATION_DAYS

File: admin/partials/digital-climate-strike-wordpress-admin-display.php [hunk 2]
```diff
@@ -10,17 +10,17 @@
...
-        $iframe_host = ( isset( $options['iframe_host'] ) && ! empty( $options['iframe_host'] ) ) ? esc_attr($options['iframe_host']) : 'https://assets.digitalclimatestrike.net';
-        $cookie_expiration_days = ( isset( $options['cookie_expiration_days'] ) && ! empty( $options['cookie_expiration_days'] ) ) ? esc_attr($options['cookie_expiration_days']) : 1;
-        $disable_google_analytics = ( isset( $options['disable_google_analytics'] ) && ! empty( $options['disable_google_analytics'] ) ) ? 1 : 0;
-        $show_close_button_on_full_page_widget = ( isset( $options['show_close_button_on_full_page_widget'] ) && ! empty( $options['show_close_button_on_full_page_widget'] ) ) ? 1 : 0;
+        $force_full_page_widget = $this->field_is_set($options, 'force_full_page_widget') ? self::ENABLE : self::DISABLE;
+        $always_show_widget = $this->field_is_set($options, 'always_show_widget') ? self::ENABLE : self::DISABLE;
+        $iframe_host = $this->field_is_set($options, 'iframe_host') ? esc_attr($options['iframe_host']) : 'https://assets.digitalclimatestrike.net';
```
Comment (dtoakley) [inline admin/partials/digital-climate-strike-wordpress-admin-display.php]: should we extract the iFrame host string to . constant as well?

File: admin/partials/digital-climate-strike-wordpress-admin-display.php [hunk 3]
```diff
@@ -10,17 +10,17 @@
...
+        $cookie_expiration_days = $this->field_is_set($options, 'cookie_expiration_days') ? esc_attr($options['cookie_expiration_days']) : self::ENABLE;
```
Comment (dtoakley) [inline admin/partials/digital-climate-strike-wordpress-admin-display.php]: As above, i think this should have it's own constant for clarity, and incase we want to change the default

File: admin/class-digital-climate-strike-wordpress-admin.php (hunk 1 above)
Comment (sturosier) [reply admin/class-digital-climate-strike-wordpress-admin.php]: Fixed.

File: admin/partials/digital-climate-strike-wordpress-admin-display.php (hunk 2 above)
Comment (sturosier) [reply admin/partials/digital-climate-strike-wordpress-admin-display.php]: Agree. Fixed as well.

File: admin/partials/digital-climate-strike-wordpress-admin-display.php (hunk 3 above)
Comment (sturosier) [reply admin/partials/digital-climate-strike-wordpress-admin-display.php]: Fixed.

**Review by dtoakley — APPROVED** [review]

**Pull request merged**